resources/views/components/supported-game-files/hash-listing.blade.php

```diff
@@ -0,0 +1,40 @@
+@props([
+    'hash' => null, // GameHash
+])
+
+<li>
+    <p>
+        @if ($hash->name)
+            <span class="font-bold">{{ $hash->name }}</span>
+        @endif
+
+        @if (!empty($hash->labels))
+            @foreach (explode(',', $hash->labels) as $label)
+                @if (empty($label))
+                    @continue;
+                @endif
+
+                @php
+                    $image = "/assets/images/labels/" . $label . '.png';
+                    $publicPath = public_path($image);
+                @endphp
+                
+                @if (file_exists($publicPath))
+                    <img class="inline-image" src="{{ asset($image) }}">
+                @else
+                    <span>[{{ $label }}]</span>
+                @endif
+            @endforeach
+        @endif
+    </p>
+
+    <div class="flex flex-col">
+        <p class="font-mono text-neutral-200 light:text-neutral-700">
+            {{ $hash->md5 }}
+        </p>
+
+        @if ($hash->patch_url)
+            <a href="{{ $hash->patch_url }}" rel="noreferrer">Download Patch File</a>
+        @endif
+    </div>
```

Comment:
I think it would be desirable for the hash and patch links to be slightly indented.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/0dfb23e8-1ffa-48ee-b2a8-1333c77c9c1b)

Or put each in their own bubble.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/3bc80db6-c5fa-4e92-82b8-0e42c245e2a8)

Right now, they all sort of run together.

Reply:
Slightly touched this up in latest:

![Screenshot 2024-04-07 at 11 31 17 AM](https://github.com/RetroAchievements/RAWeb/assets/3984985/a95d95e1-2522-4c01-8c7b-999adb48bc47)